CONTRIBUTING.md

```diff
@@ -581,18 +585,67 @@ mod tests {
 }
 ```
 
-For more detailed guidance on how to add Rust testing you can refer to the Rust
+For more detailed guidance on how to write Rust tests, you can refer to the Rust
 documentation's [guide on writing tests](https://doc.rust-lang.org/book/ch11-01-writing-tests.html).
 
-To run the Rust-space tests, do
+Rust tests are run separately from the Python tests. The easiest way to run
+them is via ``tox``, which creates an isolated venv and pre-installs ``qiskit``
+prior to running ``cargo test``:
+
+```bash
+tox -erust
+```
```

Comment:
My only concern  here is that this results in building Qiskit from source twice on every execution. I'm not sure there is a way around this since we need the python extension built to be able to call it via python in the rust tests it just is pretty slow. We should probably build in debug mode by default in tox and for rust testing by default.

Reply:
I've enabled debug mode by default for the `rust` env's package environment in 6856adbb9dc006ef0ed85f7869fd15778b470b0a.

I also documented `--skip-pkg-install` as an option for running without rebuilding Qiskit when invoking `tox -erust`, and I've explained there that this is only an option if you've already built the current working tree: 4813a7c057efe7cec1f49efe30cdbb71e4d96774